index.js

```diff
@@ -386,7 +386,7 @@
         return x != null && (
           R.type(type.prototype['@@type']) === 'String' ?
             x['@@type'] === type.prototype['@@type'] :
-            R.type(x) === R.nth(1, R.match(/^function (\w*)/, String(type)))
+            R.type(x) === R.nth(1, R.match(/function (\w*)/, String(type)))
```

Comment:
This is a strange question.
`/function (\w*)/` matchs `"\nfunction Array() {\n    [native code]\n}\n"` in IE 11. And `/^function (\w*)/` don't. This is the only reason I made this change :)